scripts/rpm-install.sh

```diff
@@ -43,25 +43,27 @@ sudo yum search maria | { grep "^maria" || true; }
 sudo yum search percona | { grep percona || true; }
 
 # setup repository for galera dependency
-if wget -q --spider https://rpm.mariadb.org/"$master_branch/$arch"; then
-  galbranch=$master_branch
+rpm_setup_bb_galera_artifacts_mirror
+
+# setup artifact repository
+rpm_setup_bb_artifacts_mirror
+
+yum_makecache
+
+if [[ -f /etc/yum.repos.d/MariaDB.repo ]]; then
+  repo_name_tmp=$(head -n1 /etc/yum.repos.d/MariaDB.repo)
+  # remove brackets
+  repo_name=${repo_name_tmp/\[/}
+  repo_name=${repo_name/\]/}
 else
-  # as long as the galera major version maps this is ok
-  galbranch=10.8
-  if ! wget -q --spider https://rpm.mariadb.org/"$galbranch/$arch"; then
-    bb_log_err "https://rpm.mariadb.org/$galbranch/$arch does not exist"
-    bb_log_err "unable to setup repository for galera"
-    exit 1
-  fi
+  bb_log_err "/etc/yum.repos.d/MariaDB.repo is missing"
 fi
-sudo sh -c "echo '[galera]
-name=galera
-baseurl=https://rpm.mariadb.org/$galbranch/$arch
-gpgkey=https://rpm.mariadb.org/RPM-GPG-KEY-MariaDB_v2
-gpgcheck=1' >/etc/yum.repos.d/galera.repo"
-
-sudo cat /etc/yum.repos.d/galera.repo
-sudo yum -y --nogpgcheck install rpms/*.rpm
+repoquery --disablerepo=* --enablerepo="${repo_name}" | xargs sudo yum -y install
```

Comment:
`yum --disablerepo= --enablerepo= ...`

Reply:
Why that, I need to get the full list of available rpm and **then** install them